### PR TITLE
Added a new article - using Lightning to migrate shard tables to TiDB

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added a new article - using Lightning to migrate shard tables to TiDB

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+下面详细介绍每一步的操作流程。
+
+### 第 1 步：用 Dumpling 导出全量数据备份
+
+如果需要导出的多个分表属于同一个上游 MySQL 实例，可以直接使用 Dumpling 的 `-f` 参数一次导出多个分表的结果。如果多个分表分布在不同的 MySQL 实例，可以使用 Dumpling 分两次导出，并将两次导出的结果放置在相同的父目录下即可。下面的例子中同时用到了上述两种方式，然后将导出的数据存放在同一父目录下。
+
+首先使用 Dumpling 从 my_db1 中导出表 table1 和 table2，如下：
+
+```
+tiup dumpling -h &lt;ip> -P &lt;port> -u root -t 16 -r 200000 -F 256MB -B my_db1 -f 'my_db1.table[12]' -o /data/my_database/
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply migrate-from-mysql-shard-merge-using-lightning.md]: 改掉了

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+
+```shell
+tiup dumpling -h &lt;ip> -P &lt;port> -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my_db2.table[34]' -o /data/my_database/
+```
+
+这样所需的全量备份数据就全部导出到了 `/data/my_database` 目录中。将所有源数据表格存储在一个目录中，是为了后续方便用 TiDB Lightning 导入。
+
+### 第 2 步：处理主键或唯一索引冲突
+
+来自多张分表的数据可能会引发主键或者唯一索引的数据冲突，因此在导入数据之前，需要结合分表逻辑对每个主键或唯一索引进行检查，还可能需要预先在下游创建表结构。更多详情，请参考 [跨分表数据在主键或唯一索引冲突处理](https://docs.pingcap.com/zh/tidb-data-migration/stable/shard-merge-best-practices#跨分表数据在主键或唯一索引冲突处理)。
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply migrate-from-mysql-shard-merge-using-lightning.md]: 和朝阳确认了下，这部分内容先删掉了。

File: migration-overview.md [hunk 3]
```diff
@@ -27,7 +27,17 @@ aliases: ['/docs-cn/dev/migration-overview/','/docs-cn/dev/data-migration-route'
 
 #### 迁移方法
 
-使用 Dumpling 导出 MySQL 的全量数据，再使用 TiDB Lightning 将全量数据导入到 TiDB，详细信息可参考[使用 Dumpling 与 TiDB Lightning 进行全量迁移](/migrate-from-mysql-dumpling-files.md)
+使用 Dumpling 导出 MySQL 的全量数据，再使用 TiDB Lightning 将全量数据导入到 TiDB，详细信息可参考[使用 Dumpling 与 TiDB Lightning 进行全量迁移](/migrate-from-mysql-dumpling-files.md)。
+
+### 使用 Dumpling 和 TiDB Lightning 合并导入分表数据
```
Comment (hfxsd) [inline migration-overview.md:32]:
```suggestion
### 将 MySQL 的分库分表合并导入到 TiDB
```

**Review by TomShawn — APPROVED** [review]

File: migration-overview.md [hunk 4]
```diff
@@ -12,46 +12,45 @@ aliases: ['/docs-cn/dev/migration-overview/','/docs-cn/dev/data-migration-route'
...
+可以使用 DM 将 1 TiB 以内的 MySQL 全量数据迁移到 TiDB。也支持同步 MySQL 的增量数据到 TiDB。详细信息请参考[使用 DM 工具从 Amazon Aurora MySQL 迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)
 
-适合 MySQL 全量数据的大小大于 1TB 的场景。该方案只能迁移全量数据，如果需要继续同步增量数据，需要再使用 TiDB Data Migration (DM) 创建增量同步任务。
+### 大于 1 TiB 的全量数据迁移
 
-#### 迁移方法
+可以使用 Dumpling 和 TiDB Lightning 迁移大于 1 TiB 的 MySQL 全量数据。详细信息请参考[使用 Dumpling 与 TiDB Lightning 进行全量迁移](/migrate-from-mysql-dumpling-files.md)。
 
-使用 Dumpling 导出 MySQL 的全量数据，再使用 TiDB Lightning 将全量数据导入到 TiDB，详细信息可参考[使用 Dumpling 与 TiDB Lightning 进行全量迁移](/migrate-from-mysql-dumpling-files.md)
+## MySQL 分库分表合并迁移到 TiDB
```
Comment (hfxsd) [inline migration-overview.md:30]:
```suggestion
<!-- 
## MySQL 分库分表合并迁移到 TiDB
```

File: migration-overview.md [hunk 5]
```diff
@@ -12,46 +12,45 @@ aliases: ['/docs-cn/dev/migration-overview/','/docs-cn/dev/data-migration-route'
...
-适合迁移 MySQL 全量数据并同步增量数据的场景，且全量数据的大小小于 1TB。如果全量数据的大小大于 1TB，建议使用 Dumpling 和 TiDB Lightning 导入全量数据后，再使用 DM 同步增量数据。
+### 1 TiB 以内的 MySQL 的分库分表合并迁移 
 
-#### 迁移方法
+对于 MySQL 全量数据小于 1 TiB、分表合并迁移的场景，可以直接使用 DM 将各 MySQL 分库分表数据合并导入到 TiDB，详细信息可参考[将 MySQL 分库分表(1 TiB以内)合并迁移到 TiDB](/migrate-shard-tables-within-1tb.md)。
 
-DM 支持将 MySQL 全量数据迁移到 TiDB，并同步 MySQL 的增量数据到 TiDB，详细信息可参考[使用 DM 工具从 Amazon Aurora MySQL 迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)
+### 大于 1 TiB 的分库分表合并迁移 
+
+对于 MySQL 全量数据大于 1 TiB、分表合并迁移的场景，可以先用 Dumpling 从多个 MySQL 数据库中导出数据，再使用 TiDB Lightning 将各分库分表数据合并导入到 TiDB。详细信息请参考[分库分表合并导入到 TiDB （大于 1 TiB）](/migrate-from-mysql-shard-merge-using-lightning.md)。
```
Comment (hfxsd) [inline migration-overview.md:43]:
```suggestion
对于 MySQL 全量数据大于 1 TiB、分表合并迁移的场景，可以先用 Dumpling 从多个 MySQL 数据库中导出数据，再使用 TiDB Lightning 将各分库分表数据合并导入到 TiDB。详细信息请参考[分库分表合并导入到 TiDB （大于 1 TiB）](/migrate-from-mysql-shard-merge-using-lightning.md)。
-->
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 6]
```diff
@@ -0,0 +1,373 @@
+---
+title: 分库分表合并导入到 TiDB
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:2]:
```suggestion
title: 分库分表合并导入到 TiDB（大于 1 TB）
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 7]
```diff
@@ -0,0 +1,373 @@
...
+summary: 使用 Dumpling 和 TiDB Lightning 合并导入分表数据到 TiDB。本文介绍的方法适用于导入数据总量大于 1 TB 的场景。
+---
+
+# 分库分表合并导入到 TiDB （大于 1 TB）
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:6]:
```suggestion
# 分库分表合并导入到 TiDB（大于 1 TB）
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 8]
```diff
@@ -0,0 +1,373 @@
...
+
+**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 实例，你可以在本地虚拟化或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程默认会占满 CPU，建议单独部署。如果条件不允许，可以和其他组件（比如 `tikv-server`）部署在同一台机器上，然后通过配置 `region-concurrency` 限制 TiDB Lightning 使用 CPU 资源。混合部署的情况下可以将其大小配置为逻辑 CPU 数的 75%，以限制 CPU 的使用。
+
+**内存和 CPU**：因为 TiDB Lightning 对计算机资源消耗较高，是资源密集型工具，建议分配 64 GB 以上的内存以及 32 核以上的 CPU，而且确保 CPU 核数和内存（GB）比为 1:2 以上，以获取最佳性能。
+
+**磁盘空间**：足够储存整个数据源的 SSD 硬盘，读取速度越快越好。
+
+### 目标 TiKV 集群的磁盘空间要求
+
+**磁盘空间**：目标 TiKV 集群必须有足够空间接收新导入的数据。除了[标准硬件配置](https://docs.pingcap.com/zh/tidb/stable/hardware-and-software-requirements)以外，目标 TiKV 集群的总存储空间必须大于 **数据源大小 ×[副本数量](https://docs.pingcap.com/zh/tidb/stable/deploy-and-maintain-faq#每个-region-的-replica-数量可配置吗调整的方法是) × 2**。例如集群默认使用 3 副本，那么总存储空间需为数据源大小的 6 倍以上。公式中的 2 倍可能难以理解，其依据是以下因素的估算空间占用：
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:43]:
```suggestion
**磁盘空间**：目标 TiKV 集群必须有足够空间接收新导入的数据。除了[标准硬件配置](/hardware-and-software-requirements.md)以外，目标 TiKV 集群的总存储空间必须大于 **数据源大小 ×[副本数量](/deploy-and-maintain-faq.md#每个-region-的-replica-数量可配置吗调整的方法是) × 2**。例如集群默认使用 3 副本，那么总存储空间需为数据源大小的 6 倍以上。公式中的 2 倍可能难以理解，其依据是以下因素的估算空间占用：
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 9]
```diff
@@ -0,0 +1,373 @@
...
+
+导入完毕后，TiDB Lightning 会自动退出。查看日志的最后 5 行中会有 `the whole procedure completed`，则表示导入成功。
+
+> **注意：**
+>
+> 无论导入成功与否，最后一行都会显示 `tidb lightning exit`。它只是表示 TiDB Lightning  正常退出，不代表任务完成。
+
+如果导入过程中遇到问题，请参见 [TiDB Lightning 常见问题](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-faq)。
+
+### 第 4 步 （可选）：增量数据的迁移
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:239]:
```suggestion
### 第 4 步（可选）：增量数据的迁移
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 10]
```diff
@@ -0,0 +1,373 @@
...
+
+**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 实例，你可以在本地虚拟化或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程默认会占满 CPU，建议单独部署。如果条件不允许，可以和其他组件（比如 `tikv-server`）部署在同一台机器上，然后通过配置 `region-concurrency` 限制 TiDB Lightning 使用 CPU 资源。混合部署的情况下可以将其大小配置为逻辑 CPU 数的 75%，以限制 CPU 的使用。
+
+**内存和 CPU**：因为 TiDB Lightning 对计算机资源消耗较高，是资源密集型工具，建议分配 64 GB 以上的内存以及 32 核以上的 CPU，而且确保 CPU 核数和内存（GB）比为 1:2 以上，以获取最佳性能。
+
+**磁盘空间**：足够储存整个数据源的 SSD 硬盘，读取速度越快越好。
+
+### 目标 TiKV 集群的磁盘空间要求
+
+**磁盘空间**：目标 TiKV 集群必须有足够空间接收新导入的数据。除了[标准硬件配置](/hardware-and-software-requirements.md)以外，目标 TiKV 集群的总存储空间必须大于 **数据源大小 ×[副本数量](/deploy-and-maintain-faq.md#每个-region-的-replica-数量可配置吗调整的方法是) × 2**。例如集群默认使用 3 副本，那么总存储空间需为数据源大小的 6 倍以上。公式中的 2 倍可能难以理解，其依据是以下因素的估算空间占用：
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:43]:
```suggestion
**磁盘空间**：目标 TiKV 集群必须有足够空间接收新导入的数据。除了[标准硬件配置](/hardware-and-software-requirements.md)以外，目标 TiKV 集群的总存储空间必须大于 **数据源大小 × [副本数量](/deploy-and-maintain-faq.md#每个-region-的-replica-数量可配置吗调整的方法是) × 2**。例如集群默认使用 3 副本，那么总存储空间需为数据源大小的 6 倍以上。公式中的 2 倍可能难以理解，其依据是以下因素的估算空间占用：
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 11]
```diff
@@ -0,0 +1,241 @@
...
+
+大量数据导入一般耗时数小时甚至数天，长时间运行的进程会有一定机率发生非正常中断。如果每次重启都从头开始，之前已成功导入的数据就会前功尽弃。为此，TiDB Lightning 提供了断点续传的功能，即使 TiDB Lightning 崩溃，在重启时仍然从断点开始继续工作。
+
+若 TiDB Lightning 因不可恢复的错误而退出，例如数据出错，在重启时不会使用断点，而是直接报错离开。为保证已导入的数据安全，必须先解决掉这些错误才能继续。你可以使用`tidb-lightning-ctl` 命令控制导入出错后的行为。该命令的选项有：
+
+* --checkpoint-error-destroy：出现错误后，让失败的表从头开始整个导入过程。
+* --checkpoint-error-ignore：如果导入表曾经出错，该命令会清除出错状态，如同错误没有发生过一样。
+* --checkpoint-remove：无论是否有出错，把表的断点清除。
+
+关于断点续传的更多信息，请参考 [TiDB Lightning 断点续传](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-checkpoints)。
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:161]:
```suggestion
关于断点续传的更多信息，请参考 [TiDB Lightning 断点续传](/tidb-lightning/tidb-lightning-checkpoints.md)。
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 12]
```diff
@@ -0,0 +1,241 @@
...
+
+#### 后端模式
+
+在启动 TiDB Lightning 的时候请选择合适的后端模式。
+
+* 如果导入的目标集群为 v4.0 或更新版本，请优先考虑使用 Local-backend 模式。Local-backend 部署更简单并且性能高于其他模式。
+* 如果目标集群为 v3.x 或更旧版本，则建议使用 Importer-backend 模式。
+* 如果需要导入的集群为生产环境线上集群，或需要导入的表中已包含有数据，则可以使用 TiDB-backend 模式。但由于该模式导入速度较慢，不适合本文介绍的大量数据迁移场景。
+
+默认使用 Local-backend 模式。本文档的示例中采用了 Local-backend 模式。更多信息请参考 [TiDB Lightning 后端](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:136]:
```suggestion
默认使用 Local-backend 模式。本文档的示例中采用了 Local-backend 模式。更多信息请参考 [TiDB Lightning 后端](/tidb-lightning/tidb-lightning-backends.md)。
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 13]
```diff
@@ -0,0 +1,241 @@
...
+
+* SELECT
+* UPDATE
+* ALTER
+* CREATE
+* DROP
+
+一般情况下，推荐使用配置项 `checksum = true`，所以 TiDB Lightning 还需要有下游 TiDB admin 用户权限。
+
+更多相关权限的说明，请参考[TiDB Lightning 对下游数据库的账号权限要求是怎样的？](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-faq#tidb-lightning-对下游数据库的账号权限要求是怎样的)
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:70]:
```suggestion
更多相关权限的说明，请参考[TiDB Lightning 对下游数据库的账号权限要求是怎样的？](/tidb-lightning/tidb-lightning-faq.md#tidb-lightning-对下游数据库的账号权限要求是怎样的)
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 14]
```diff
@@ -0,0 +1,241 @@
...
+
+### 第 3 步：导入成功
+
+导入完毕后，TiDB Lightning 会自动退出。查看日志的最后 5 行中会有 `the whole procedure completed`，则表示导入成功。
+
+> **注意：**
+>
+> 无论导入成功与否，最后一行都会显示 `tidb lightning exit`。它只是表示 TiDB Lightning  正常退出，不代表任务完成。
+
+如果导入过程中遇到问题，请参见 [TiDB Lightning 常见问题](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-faq)。
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:237]: ditto

File: migration-overview.md [hunk 15]
```diff
@@ -12,46 +12,47 @@ aliases: ['/docs-cn/dev/migration-overview/','/docs-cn/dev/data-migration-route'
...
-- [使用 Dumpling 和 TiDB Lightning 迁移全量数据](#使用-dumpling-和-tidb-lightning-迁移全量数据)
-- [使用 DM 迁移数据](#使用-dm-迁移数据)
+- 1 TiB 以内的全量数据迁移
+- 大于 1 TiB 的全量数据迁移
 
-### 使用 Dumpling 和 TiDB Lightning 迁移全量数据
+### 1 TiB 以内的全量数据迁移
 
-#### 适合场景
+可以使用 DM 将 1 TiB 以内的 MySQL 全量数据迁移到 TiDB。也支持同步 MySQL 的增量数据到 TiDB。详细信息请参考[使用 DM 工具从 Amazon Aurora MySQL 迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)
```
Comment (TomShawn) [inline migration-overview.md:24]:
```suggestion
可以使用 DM 将 1 TiB 以内的 MySQL 全量数据迁移到 TiDB。也支持同步 MySQL 的增量数据到 TiDB。详细信息请参考[使用 DM 工具从 Amazon Aurora MySQL 迁移](https://docs.pingcap.com/zh/tidb-data-migration/stable/migrate-from-mysql-aurora)
```

File: migrate-from-mysql-shard-merge-using-lightning.md (hunk 14 above)
Comment (hfxsd) [inline migrate-from-mysql-shard-merge-using-lightning.md:237]:
```suggestion
如果导入过程中遇到问题，请参见 [TiDB Lightning 常见问题](/tidb-lightning/tidb-lightning-faq)。
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 16]
```diff
@@ -0,0 +1,241 @@
...
+使用 TiDB Lightning 快速合并导入的原理如下图所示。
+
+![使用 Dumpling 和 TiDB Lightning 合并导入分表数据](/media/shard-merge-using-lightning.png)
+
+在这个示例中，假设有两个数据库 my_db1 和 my_db2 ，使用 Dumpling 分别从 my_db1 中导出 table1 和 table2 两个表，从 my_db2 中导出 table3 和 table4 两个表，然后再用 TiDB Lighting 把导出的 4 个表合并导入到下游 TiDB 中的同一个库 my_db 的同一个表格 table5 中。虽然 Dumpling 可以导出 MySQL 实例中的所有数据库，本文仅导出部分数据作为示例。
+
+关于 Dumpling 和 TiDB Lightning 的更多介绍，请参考：
+
+* [Dumpling](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview)
+* [TiDB Lightning](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview)
```
Comment (hfxsd) [inline migrate-from-mysql-shard-merge-using-lightning.md:21]:
```suggestion
* [Dumpling](/dumpling-overview.md)
* [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)
```

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 17]
```diff
@@ -0,0 +1,241 @@
...
+
+**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 实例，你可以在本地虚拟化或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程默认会占满 CPU，建议单独部署。如果条件不允许，可以和其他组件（比如 `tikv-server`）部署在同一台机器上，然后通过配置 `region-concurrency` 限制 TiDB Lightning 使用 CPU 资源。混合部署的情况下可以将其大小配置为逻辑 CPU 数的 75%，以限制 CPU 的使用。
+
+**内存和 CPU**：因为 TiDB Lightning 对计算机资源消耗较高，是资源密集型工具，建议分配 64 GB 以上的内存以及 32 核以上的 CPU，而且确保 CPU 核数和内存（GB）比为 1:2 以上，以获取最佳性能。
+
+**磁盘空间**：足够储存整个数据源的 SSD 硬盘，读取速度越快越好。
+
+### 目标 TiKV 集群的磁盘空间要求
+
+**磁盘空间**：目标 TiKV 集群必须有足够空间接收新导入的数据。除了[标准硬件配置](/hardware-and-software-requirements.md)以外，目标 TiKV 集群的总存储空间必须大于 **数据源大小 ×[副本数量](/faq/deploy-and-maintain-faq.md#每个 region 的 replica 数量可配置吗？调整的方法是？) × 2**。例如集群默认使用 3 副本，那么总存储空间需为数据源大小的 6 倍以上。公式中的 2 倍可能难以理解，其依据是以下因素的估算空间占用：
```
Comment (TomShawn) [inline migrate-from-mysql-shard-merge-using-lightning.md:43]:
```suggestion
**磁盘空间**：目标 TiKV 集群必须有足够空间接收新导入的数据。除了[标准硬件配置](/hardware-and-software-requirements.md)以外，目标 TiKV 集群的总存储空间必须大于 **数据源大小 × [副本数量](/faq/deploy-and-maintain-faq.md#每个-region-的-replica-数量可配置吗调整的方法是) × 2**。例如集群默认使用 3 副本，那么总存储空间需为数据源大小的 6 倍以上。公式中的 2 倍可能难以理解，其依据是以下因素的估算空间占用：
```

**Review by TomShawn — APPROVED** [review]

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 18]
```diff
@@ -0,0 +1,241 @@
+---
+title: 分库分表合并导入到 TiDB（大于 1 TB）
```
Comment (sunzhaoyang) [inline migrate-from-mysql-shard-merge-using-lightning.md:2]: TB 统一 改为  TiB 吧

File: migrate-from-mysql-shard-merge-using-lightning.md [hunk 19]
```diff
@@ -0,0 +1,241 @@
...
+
+一般情况下，推荐使用配置项 `checksum = true`，所以 TiDB Lightning 还需要有下游 TiDB admin 用户权限。
+
+更多相关权限的说明，请参考[TiDB Lightning 对下游数据库的账号权限要求是怎样的？](/tidb-lightning/tidb-lightning-faq.md#TiDB Lightning 对下游数据库的账号权限要求是怎样的？)
+
+<!--
+### 部署 Dumpling 和 TiDB Lightning
+
+* [使用 TiUP 部署 TiDB Lighting](https://github.com/pingcap/docs/pull/6144/files#diff-53a937b1281c0fcb1ca972172e4c1b31a97af54bc5414e3186a648ea9eef6e23)
+* [使用 TiUP 部署 Dumping](https://github.com/pingcap/docs/pull/6144/files#diff-53a937b1281c0fcb1ca972172e4c1b31a97af54bc5414e3186a648ea9eef6e23)
```
Comment (sunzhaoyang) [inline migrate-from-mysql-shard-merge-using-lightning.md:76]: 这里为啥注释掉了？不写安装过程了吗

**Comment by sunzhaoyang** [comment]
rest LGTM

File: migrate-from-mysql-shard-merge-using-lightning.md (hunk 19 above)
Comment (hfxsd) [reply migrate-from-mysql-shard-merge-using-lightning.md:76]: 这个功能不是还没上线吗？等上线后再把链接加上。

File: migrate-from-mysql-shard-merge-using-lightning.md (hunk 18 above)
Comment (hfxsd) [reply migrate-from-mysql-shard-merge-using-lightning.md:2]: Done。

**Pull request closed without merging**